lib/daru/core/group_by.rb

```diff
@@ -298,6 +312,31 @@ def apply_method method_type, method
         Daru::DataFrame.new(rows.transpose, index: index, order: order)
       end
 
+      # Do the `method` (`method` can be :sum, :mean, :std, :median, etc or
+      # lambda/function/Proc), on the column.
+      def apply_method_on_colmns colmns, method
+        rows = []
+        @keys.each do |indexes|
+          slice = @df[colmns][*indexes]
```

Comment:
slicing the resultant dataframe `@df` of the group_by method, using multi index `indexes`.